backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/theme/BpkTheme.kt

```diff
@@ -75,11 +76,23 @@ object BpkTheme {
 
   val typography: BpkTypography
     @Composable
-    get() = LocalBpkTypography.current
+    get() = if (LocalInspectionMode.current) {
+      // when in preview mode return a default typography object to ensure previews work
+      // without wrapping it in another composable
+      BpkTypography(defaultFontFamily = FontFamily.SansSerif)
+    } else {
+      LocalBpkTypography.current
+    }
 
   val colors: BpkColors
     @Composable
-    get() = LocalBpkColors.current
+    get() = if (LocalInspectionMode.current) {
+      // when in preview mode return a default colour object to ensure previews work
+      // without wrapping it in another composable
+      BpkColors.light()
```

Comment:
We can actually check the UI mode here and return dark if needed. DM can be enabled with just an annotation.

Reply:
good shout - that'll also help in the future when the different device configurations come along